webrtc-stats.html

```diff
@@ -430,10 +430,14 @@
             <dl data-link-for="RTCRTPStreamStats" data-dfn-for="RTCRTPStreamStats" class=
             "dictionary-members">
               <dt>
-                <dfn><code>ssrc</code></dfn> of type <span class=
-                "idlMemberType"><a>DOMString</a></span>
+                <dfn><code>ssrc</code></dfn> of type <span class="idlMemberType"><a>unsigned long
+                </a></span>
               </dt>
-              <dd></dd>
+              <dd>
+                <p>
+                  32-bit integer value per [[RFC3550]].
```

Comment:
Description updated.
(The spec says "identified by a 32-bit numeric SSRC identifier carried in the RTP header", based on it it is not clear to me whether it should be signed, unsigned or just a series of bits, but its appendix has sample code that use uint32_t :) )